packages/forma-36-react-components/src/index.ts

```diff
@@ -19,6 +19,20 @@ export type {
 } from '@contentful/f36-core';
 export { Spinner } from '@contentful/f36-spinner';
 export type { SpinnerProps } from '@contentful/f36-spinner';
+export {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+} from '@contentful/f36-table';
```

Comment:
💭 should we introduce a wrapper for `<tfoot>` as well, I don't know if we have any table footers we use across our apps?

Reply:
Not sure if we need a table footer in Contentful for now. We don't have any number-heavy tables that would benefit from it. What would be an example use?